### PR TITLE
user_external: Update some links

### PR DESCRIPTION
**Pull request by MariusBluem**
Signed-off-by: Marius Blüm <marius@lineone.io>

### CODE REVIEW

File: user_external/appinfo/info.xml [hunk 1]
```diff
@@ -6,19 +6,18 @@
...
-	<bugs>https://github.com/owncloud/apps/issues</bugs>
-	<repository type="git">https://github.com/owncloud/apps.git</repository>
+  	<admin>https://docs.nextcloud.com/server/12/admin_manual/configuration_user/user_auth_ftp_smb_imap.html</admin>
+  </documentation>
+	<website>https://github.com/nextcloud/apps/tree/master/user_external</website>
+	<bugs>https://github.com/nextcloud/apps/issues</bugs>
+	<repository type="git">https://github.com/nextcloud/apps.git</repository>
 	<dependencies>
-		<nextcloud min-version="11" max-version="11" />
+		<nextcloud min-version="12" max-version="12" />
```
Comment (icewind1991) [inline user_external/appinfo/info.xml]: you can probably leave the min version at 11

Comment (MariusBluem) [inline user_external/appinfo/info.xml]: But `stable11` has its own branch ... `master` is 12.0, isn't? 🙈

Comment (MariusBluem) [inline user_external/appinfo/info.xml]: ping @icewind1991

Comment (icewind1991) [inline user_external/appinfo/info.xml]: ok, lets do it like this

**Pull request merged**